src/orion/storage/base.py

```diff
@@ -19,6 +19,42 @@
 log = logging.getLogger(__name__)
 
 
+def get_experiment_uid(experiment=None, uid=None, force_uid=True):
+    """Return experiment uid either from `experiment` or directly uid.
+
+    Delete matching experiments from the database
```

Comment:
Shouldn't this be in `delete_experiment` ?

Reply:
Indeed. I corrected the docstring in #428